src/bench_alkahest.rs

```diff
@@ -0,0 +1,46 @@
+use criterion::{black_box, Criterion};
+use alkahest::{
+    Pack, Schema, Unpacked,
+};
+
+pub fn bench<T, D>(name: &'static str, c: &mut Criterion, data: D, read: fn(Unpacked<'_, T>))
+where
+    T: Schema,
+    D: Pack<T> + Copy,
+{
+    let mut group = c.benchmark_group(format!("{}/alkahest", name));
+
+    // This leak is required because of ICE in rustc if `R` has bound like this `for<'a> Fn(Unpacked<'a, T>)`.
+    let mut buffer = vec![0u64; 1 << 1 << 22];
```

Comment:
I would prefer `1 << 1 << 22` be moved into a constant named `BUFFER_LEN` with the size explicitly set to `8_388_608`. It will make it easier to understand relative to the other benchmarks.